src/main/java/seedu/partyplanet/storage/JsonAdaptedPerson.java

```diff
@@ -115,12 +115,14 @@ public Person toModelType() throws IllegalValueException {
         if (birthday.equals(Birthday.EMPTY_BIRTHDAY_STRING)) {
             modelBirthday = Birthday.EMPTY_BIRTHDAY;
         }
-        try {
-            modelBirthday = new Birthday(birthday);
-        } catch (DateTimeException err) { // birthday year exceeds current year
-            throw new IllegalValueException((Birthday.MESSAGE_YEAR_CONSTRAINTS));
-        } catch (IllegalArgumentException err) { // date in wrong format
-            throw new IllegalValueException(Birthday.MESSAGE_CONSTRAINTS);
+        else {
+            try {
+                modelBirthday = new Birthday(birthday);
+            } catch (DateTimeException err) { // birthday year exceeds current year
+                throw new IllegalValueException((Birthday.MESSAGE_YEAR_CONSTRAINTS));
+            } catch (IllegalArgumentException err) { // date in wrong format
+                throw new IllegalValueException(Birthday.MESSAGE_CONSTRAINTS);
```

Comment:
Swap error message
```suggestion
            } catch (DateTimeException err) { // date in wrong format
                throw new IllegalValueException((Birthday.MESSAGE_CONSTRAINTS));
            } catch (IllegalArgumentException err) { // birthday year exceeds current year
                throw new IllegalValueException(Birthday.MESSAGE_YEAR_CONSTRAINTS);
```

Reply:
Screenshot of what @garyljj is referring to:

![image](https://user-images.githubusercontent.com/28663438/111090270-26dd5880-856a-11eb-845e-1a4d6867f943.png)